antenna-core/src/main/java/org/eclipse/sw360/antenna/util/XmlSettingsReader.java

```diff
@@ -49,7 +50,7 @@
      */
     public XmlSettingsReader(String xmlString) throws ParserConfigurationException, SAXException, IOException {
         DocumentBuilder dBuilder = initDocumentBuilder();
-        this.doc = dBuilder.parse(new ByteArrayInputStream(xmlString.getBytes("UTF-8")));
+        this.doc = dBuilder.parse(new ByteArrayInputStream(xmlString.getBytes(StandardCharsets.UTF_8)));
```

Comment:
And a third way to express UTF-8 :-)

Reply:
Good catch. This was an auto-IDE replacement. 

I guess this is the best way to express it, because there is no magic string involved and thus there is no spelling error possible. I'll change all occurrences to this way here.